packages/plugins/connection-manager/extension.ts

```diff
@@ -344,6 +345,38 @@ export default class ConnectionManagerPlugin implements IExtensionPlugin {
     return commands.executeCommand('vscode.open', file);
   }
 
+  private ext_openResults = async (arg: IQueryOptions | Uri = {}) => {
+    let opt: IQueryOptions = {};
+    if (arg instanceof Uri) {
+      // if clicked on editor title actions
+      const view = this.resultsWebview.getActiveView();
+      if (!view) {
+        throw 'Can\'t find active results view';
+      }
+      const state = await view.getState();
+      const activeResult = state.resultTabs[state.activeTab];
+      opt = {
+        requestId: activeResult.requestId,
+        resultId: activeResult.resultId,
+        baseQuery: activeResult.baseQuery,
+        connId: activeResult.connId,
+      };
+    } else if (arg.requestId) {
+      // used context menu inside of a view
+      const { ...rest } = arg;
+      opt = rest;
+    }
+
+    if (!opt || !opt.requestId) throw "Can't find active results view";
+
+    var filename = file({
```

Comment:
Can we make it open both JSON and CSV as we do here: https://github.com/mtxr/vscode-sqltools-pro/blob/f7b12ff472fb27eddcc5f05ba150242c15fdf394/packages/plugins/connection-manager/extension.ts#L325

Reply:
Well, I had "Open with Excel" in my mind, that is why only `csv` option. I will update this part to add `json` option. 

Also, I think `json` should open in vscode and `csv` should open in external program (i.e. Excel etc)?